.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
-* @Kelvinrr
\ No newline at end of file
+* @Kelvinrr
+* @amystamile-usgs
```

Comment:
@Kelvinrr I guess I need write access to the repo?

Reply:
The point of adding to this file is to give you write access, so that error is weird. I don't have access to the settings. I think all that is handled by the condo forge bots